### PR TITLE
Deadbushes drop sticks

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: src/Blocks/BlockDeadBush.h [hunk 1]
```diff
@@ -42,6 +42,21 @@ class cBlockDeadBushHandler :
 		UNUSED(a_Meta);
 		return 0;
 	}
+
+	virtual void OnDestroyedByPlayer(cChunkInterface & a_ChunkInterface, cWorldInterface & a_WorldInterface, cPlayer * a_Player, int a_BlockX, int a_BlockY, int a_BlockZ) override
```
Comment (Pokechu22) [inline src/Blocks/BlockDeadBush.h]: Is this the best method to override?  [`BlockTallGrass` uses `DropBlock`](https://github.com/cuberite/cuberite/blob/5effe2cc7916e067e6f6579a73c1101a4e58dd63/src/Blocks/BlockTallGrass.h#L36).  That would also allow a plugin callback:

```a_BlockPluginInterface.CallHookBlockToPickups(a_Digger, a_BlockX, a_BlockY, a_BlockZ, m_BlockType, Meta, Drops);```

Comment (madmaxoft) [inline src/Blocks/BlockDeadBush.h]: We have too many similar methods to override, and then it leads to situations like this. It seems like the `DropBlock` method would be preferred for this indeed.

Comment (Bond-009) [inline src/Blocks/BlockDeadBush.h]: I will change it when I have time.

**Comment by Bond-009** [comment]
Do I need to change anything else?

**Comment by madmaxoft** [comment]
Looks good. Sorry for the late reply.

**Pull request merged**